### ISSUE TITLE
Clarify doc to create labels from scratch

### ISSUE THREAD

**Issue by jcohenadad** [spinalcordtoolbox/manual-correction#85]
The [current doc](https://github.com/spinalcordtoolbox/manual-correction?tab=readme-ov-file#3-usage) is great to use when starting from YML file, but what if no YML file is present? (eg: labeling from scratch)

The [example section](https://github.com/spinalcordtoolbox/manual-correction?tab=readme-ov-file#4-examples) does include examples of starting from scratch, but we don't intuitively find it.

**Comment by valosekj** [comment]
Good point! Labeling from scratch is automatically done if the output path does not contain any labels:

https://github.com/spinalcordtoolbox/manual-correction/blob/5950172fb2a3a83fab9af755e19c5795d203a128/utils.py#L236

I agree that this is not well documented-- assigning myself.
